src/components/steps/ingredients/ingredients.tsx

```diff
@@ -111,31 +174,43 @@ const Ingredients = () => {
           </Button>
         </form>
 
-        <Stack
-          gap={'.5rem'}
+        <Box
           className="list"
-          sx={{ flexWrap: 'wrap', overflow: 'auto' }}
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            overflow: 'auto',
+          }}
         >
-          {ingredientsList?.map((ingredient, index) => (
-            <Chip
-              onDelete={() => handleRemoveFromIngredients(index)}
-              key={index}
-              label={ingredient}
-              sx={{ width: 'fit-content', padding: '0 .5rem' }}
-            ></Chip>
+          <Typography>Ingredients:</Typography>
+          {components?.map((component) => (
+            <Typography
```

Comment:
What I would do here is to destructure the component variable and use its fields directly below. components?.map(({ id, raw_text }) => ...